src/database/models/bookedaccommodations.js

```diff
@@ -0,0 +1,26 @@
+module.exports = (sequelize, DataTypes) => {
+  const bookedaccommodations = sequelize.define('bookedaccommodations', {
+    userid: DataTypes.INTEGER,
+    accommodationid: DataTypes.INTEGER,
+    roomtype: DataTypes.STRING,
+    numberofbeds: DataTypes.INTEGER,
```

Comment:
wouldn't be enough to just have a roomID instead of these detailed data (i.e. roomType and numberOfBeds)?